internal/transport/keepalive_test.go

```diff
@@ -225,8 +214,8 @@ func (s) TestKeepaliveServerClosesUnresponsiveClient(t *testing.T) {
 func (s) TestKeepaliveServerWithResponsiveClient(t *testing.T) {
 	serverConfig := &ServerConfig{
 		KeepaliveParams: keepalive.ServerParameters{
-			Time:    1 * time.Second,
-			Timeout: 1 * time.Second,
+			Time:    100 * time.Millisecond,
+			Timeout: 10 * time.Millisecond,
```

Comment:
We can safely set this ~100ms and have the same effect with hopefully less flakiness.